tests/test-inference.js

```diff
@@ -638,6 +640,23 @@ var tests = [
           verbose: false
         }
       },
+      tensorLaplace: {
+        std: { tol: 0.15 },
+        args: {
+          stepSize: 0.002,
```

Comment:
`stepSize` is a property of the optimization method, so you'll need something like `{optMethod: {adam: {stepSize: 0.002}}}` here.